<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="Power_LCD">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>300</width>
      <height>150</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Time_LCD">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>301</width>
      <height>150</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Freq_LCD">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>300</width>
      <height>150</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Частота (Гц)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время (мин)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QComboBox" name="com_select">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>СТАРТ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>СТОП</string>
    </property>
   </widget>
   <widget class="QSlider" name="Freq_set">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>410</y>
      <width>380</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimum">
     <number>25</number>
    </property>
    <property name="maximum">
     <number>65</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Time_set">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>380</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="com_apply">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_zero">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>110</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>УСТАНОВКА НУЛЯ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Сила (Н)</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="com_close">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_1_min">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>-1 мин</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_1_Hz">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>420</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>-1 Гц</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_1_min">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>240</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>+1 мин</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_1_Hz">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>420</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>+1 Гц</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>Time_LCD</zorder>
   <zorder>Power_LCD</zorder>
   <zorder>Freq_LCD</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>com_select</zorder>
   <zorder>start_button</zorder>
   <zorder>stop_button</zorder>
   <zorder>Freq_set</zorder>
   <zorder>Time_set</zorder>
   <zorder>com_apply</zorder>
   <zorder>set_zero</zorder>
   <zorder>com_close</zorder>
   <zorder>minus_1_min</zorder>
   <zorder>minus_1_Hz</zorder>
   <zorder>plus_1_min</zorder>
   <zorder>plus_1_Hz</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
